cmake/Modules/SystemInformation.cmake

```diff
@@ -175,9 +185,9 @@ if(UNIX)
     string(TOLOWER ${DISTRO_ID} DISTRO_ID)
     # Now mangle some names
     set(LINUX_NAME "${DISTRO_ID}_${DISTRO_RELEASE}")
-    if(DISTRO_ID MATCHES "fedora|mandriva|suse|opensuse|centos|scientific")
+    if(DISTRO_ID MATCHES "fedora|mandriva|suse|opensuse|centos|scientific|RHEL")
       set(SPECIFIC_SYSTEM_PREFERED_CPACK_GENERATOR "RPM")
-    endif(DISTRO_ID MATCHES "fedora|mandriva|suse|opensuse|centos|scientific")
+    endif(DISTRO_ID MATCHES "fedora|mandriva|suse|opensuse|centos|scientific|RHEL")
     if(DISTRO_ID MATCHES "debian|ubuntu")
       set(SPECIFIC_SYSTEM_PREFERED_CPACK_GENERATOR "DEB")
     endif(DISTRO_ID MATCHES "debian|ubuntu")
```

Comment:
This is a really useful change, thanks. I wonder if the module file is even needed anymore and the system one is appropriate...